pkg/standalone/standalone.go

```diff
@@ -52,18 +52,23 @@ const (
 	pubSubYamlFileName     = "pubsub.yaml"
 	stateStoreYamlFileName = "statestore.yaml"
 
-	// used when DAPR_DEFAULT_IMAGE_REGISTRY is not set.
+	// accepted DAPR_DEFAULT_IMAGE_REGISTRY values.
 	dockerContainerRegistryName = "dockerhub"
-	daprDockerImageName         = "daprio/dapr"
-	redisDockerImageName        = "redis:6"
-	zipkinDockerImageName       = "openzipkin/zipkin"
+	githubContainerRegistryName = "ghcr"
+
+	// used when DAPR_DEFAULT_IMAGE_REGISTRY is not set.
+	daprDockerImageName   = "docker.io/daprio/dapr"
+	redisDockerImageName  = "docker.io/redis:6"
+	zipkinDockerImageName = "docker.io/openzipkin/zipkin"
 
 	// used when DAPR_DEFAULT_IMAGE_REGISTRY is set as GHCR.
-	githubContainerRegistryName = "ghcr"
-	ghcrURI                     = "ghcr.io/dapr"
-	daprGhcrImageName           = "dapr"
-	redisGhcrImageName          = "3rdparty/redis:6"
-	zipkinGhcrImageName         = "3rdparty/zipkin"
+	dockerURI = "docker.io"
+	ghcrURI   = "ghcr.io"
+
+	// used when DAPR_DEFAULT_IMAGE_REGISTRY is set as GHCR or image-registry flag is set.
+	daprGhcrImageName   = "dapr/dapr"
```

Comment:
Should we make this also FQDN and have separate for image-registry ?

Reply:
the image name is concatenated and made FQDN when ghcr is selected or a private registry is given ...